src/ocm/components/clusters/ClusterPage.tsx

```diff
@@ -37,8 +37,13 @@ type MatchParams = {
   clusterId: string;
 };
 
-const ClusterPage: React.FC<RouteComponentProps<MatchParams>> = ({ match }) => {
-  const { clusterId } = match.params;
+const ClusterPageGeneric: React.FC<{ clusterId: string; showBreadcrumbs: boolean }> = ({
```

Comment:
I don't see any changes in the Router. Is this change safe?

Reply:
Yes, see line 188:
```
export const ClusterPage
```

Reply:
and 
https://github.com/openshift-assisted/assisted-ui-lib/pull/1879/files/6b08f09391dbb60f105282501c9f61a409835430#diff-405056fa39d9616ee42a0921614834f7e59ea052d79936ca0b82781049dfc3a2R6

Reply:
`showBreadcrumbs` can be optional with a false default value